File: If-Else Programming/003. Determine Character Properties/Java/solution.java
import java.io.*;
public class Main
{
    public static void main(String args[]) throws IOException
    {
        char key;
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Press any Key: ");
        key = (char) bf.read();

        if(key >= 97 && key <= 123) {
            System.out.println("Lowercase"); 
        }
        else if(key >= 65 && key <= 96) {
            System.out.println("Uppercase");
        }
        else if(key >= 48 && key <= 57) {
            System.out.println("Digit");
        }
        else {
            System.out.println("Symbol");
        }
    }
}

File: If-Else Programming/003. Determine Character Properties/C/solution.c
#include <stdio.h>

void main() {
    char a;
    printf("Press Any Key : ");
    scanf("%c", & a);

    if (a >= 65 && a <= 90) {
        printf("Uppercase");
    } 
    else if (a >= 97 && a <= 122) {
        printf("Lowercase");
    } 
    else if (a >= 48 && a <= 57) {
        printf("Digit");
    } 
    else {
        printf("Symbol");
    }
}
